.github/workflows/ci_ubuntu.yaml

```diff
@@ -94,6 +94,8 @@ jobs:
       - name: Run CodeQL analysis
         uses: github/codeql-action/analyze@v2
         if: startsWith(matrix.libt_version, 2) && (matrix.qbt_gui == 'GUI=ON') && startsWith(matrix.qt_version, 6)
+        with:
+          category: ${{ github.base_ref || github.ref_name }}
```

Comment:
Could you explain it briefly?

Reply:
The codeql needs a 'profile' to match the scan results. Previously it was automatically determined by using the variables from the CI matrix. This caused problems because we occasionally bump library versions and in a way, created a new 'profile' but codeql wouldn't know what to do with the old profile and it was confused. Now it is given a fixed value which is the git target branch (such as `master` or `v4_6_x`).
Also note that `github.base_ref` is not always present so `github.ref_name` is also used and they will refer to the same target branch.